docs/DesignDocs/system_ocall_opt_in.md

```diff
@@ -0,0 +1,120 @@
+# Explicit Enclave Opt-in to system ocalls
+
+System ocalls are ocalls that are part of openenclave core libraries. These are generally not
+called by the enclave developer but instead are called by other functions internal to OE SDK.
+
+Because ocalls have potential security impact, and enclave developer should be able to opt
```

Comment:
I would suggest consistency with [Intel documentation](https://software.intel.com/en-us/node/702976) casing on this term: OCall, ECall